plugins/gcloud/gcloud.plugin.sh

```diff
@@ -0,0 +1,49 @@
+#! bash oh-my-bash.module
+# gcloud.plugin.sh
+# Author: Antonino Cangialosi (github.com/ninoCan)
+# Fork of oh-my-bash gcloud plugin
```

Comment:
Did you mean the following?

```suggestion
# Fork of oh-my-zsh gcloud plugin
```

Reply:
Yes, I did. It was most likely overwritten by a batch substitution!